.gitsplit.yml

```diff
@@ -0,0 +1,49 @@
+# Path to a cache directory Used to speed up the split over time by reusing git's objects
+cache_url: "/cache/gitsplit"
+
+# Packages to split
+splits:
+  -
+    prefix: "src/batch"
+    target: "https://${GH_TOKEN}@github.com/yokai-php/batch-documentation.git"
```

Comment:
I'm not very familiar with git split.
I think I understand the config of the other packages but I have a doubt about "src/batch" and "docs".
Are the targets right ?

Reply:
Yeah, it's totally messed up
I figured out that right after merging
It's fixed directly on 0.x
But thanks for notifying